certbot/main.py

```diff
@@ -48,72 +48,52 @@
 logger = logging.getLogger(__name__)
 
 
-def _suggest_donation_if_appropriate(config, action):
+def _suggest_donation(config):
```

Comment:
Before it was conditional based on action and verb, now it's only conditional based on "are we a test case". Which is less conditional but still conditional -- I can change it back.